packages/frontend-2/lib/viewer/composables/activity.ts

```diff
@@ -174,7 +176,7 @@ export function useViewerUserActivityTracking(params: {
     const incomingSessionId = event.sessionId
 
     if (sessionId.value === incomingSessionId) return
-    if (status === ViewerUserActivityStatus.Disconnected) {
+    if (!isEmbedEnabled && status === ViewerUserActivityStatus.Disconnected) {
```

Comment:
This feels very inefficient - why not just disable the subscription altogether if we're in embed mode? Instead of still receiving and processing all updates, just doing nothing if `isEmbedEnabled`